modules/op-installing-and-deploying-tekton-hub-on-an-openshift-cluster.adoc

```diff
@@ -0,0 +1,105 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="op-installing-and-deploying-tekton-hub-on-an-openshift-cluster_{context}"]
+= Installing and deploying {tekton-hub} on an OpenShift cluster
```

Comment:
If you refer to the OpenShift product, it helps if you use the {product-title} attribute instead of "OpenShift." This way, our upstream docs can be vendor-neutral and not Red Hat-branded.